### DISCUSSION TITLE
When falil,continue play with nf mod

### DISCUSSION THREAD

**Discussion by guanjiao23456** [ppy/osu#28381] [Ideas] [answered]
when play without nf or autoplay,and fail,player can choose to continur playing with nf mod,similar to multiplay,when fail the game will not finish ,but ontinue
than , compute player's score and PP as nf mod

**Comment by guanjiao23456** [comment]
or create a new mod as i mean,"choosable not fail"

**Comment by BlockHead007** [comment, marked as answer]
#25921
